include/ddc/uniform_discretization.hpp

```diff
@@ -51,6 +55,8 @@ class UniformDiscretization
      * @param a the coordinate of a first real point (will have mesh coordinate 0)
      * @param b the coordinate of the second real point (will have mesh coordinate `n-1`)
      * @param n the number of points to map the segment \f$[a, b]\f$ including a & b
+     * 
+     * @deprecated use the version accepting a vector for n instead
      */
     constexpr UniformDiscretization(rcoord_type a, rcoord_type b, std::size_t n)
```

Comment:
Should we mark it `[[deprecated]]` ?

Reply:
As long as our only user is Voice, I even think we should remove it ASAP